pymeasure/instruments/mksinst/mks974b.py

```diff
@@ -195,13 +163,12 @@ def id(self):
 
     unit = Instrument.control(
         "U?", "U!%s",
-        """Control pressure unit used for all pressure readings from the instrument""",
+        """Control pressure unit used for all pressure readings from the instrument.
+
+        Allowed units are Unit.Torr, Unit.mbar, Unit.Pa.""",
         validator=strict_discrete_set,
         map_values=True,
-        values={"Torr": "TORR",
-                "mBar": "MBAR",
-                "Pascal": "PASCAL",
-                },
+        values={u: u.value for u in Unit},
```

Comment:
Interesting solution.
In https://github.com/pymeasure/pymeasure/issues/736#issuecomment-1318453826 I collected several methods to use enums with some comments.
I added your solution there.

Reply:
ok, i did not see this issue. Would be nice to have the best/recommended (if there is any) way in the documentation. Let me know if here you want it any different.

Reply:
> Would be nice to have the best/recommended (if there is any) way in the documentation.

That was exactly the intend of that issue, to write a documentation, but we did not find a perfect solution.

Keep it as is.